.github/workflows/labeler.yml

```diff
@@ -0,0 +1,13 @@
+name: "Pull Request Labeler"
+on:
+    schedule:
+        # * is a special character in YAML so you have to quote this string
+        - cron:  '*/5 * * * *'
+
+jobs:
+    triage:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: mshima/labeler@v2
```

Comment:
why is it your own action ? did you fork the project ? in this case, we can host it under jhipster organization

Reply:
Sure.
The changes I’ve made is https://github.com/actions/labeler/pull/49.
I don’t have permission to create new repositories.
Can you clone my?
 If you clone upstream, then you have to copy the dependencies too.